src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -105,6 +105,13 @@ public sealed class DiagnosticLogManager : AgentService, IDiagnosticLogManager
                 File.Copy(agentLogFile, destination);
             }
 
+            // Copy event logs for windows machines
+            if (PlatformUtil.RunningOnWindows)
+            {
+                executionContext.Debug("Dumping event viewer logs.");
```

Comment:
Nit: could it make sense to clarify that this is specifically for the current job - in the message and method name?